### ISSUE TITLE
Cursor placement not possible using pointer in <input>

### ISSUE THREAD

**Issue by d53dave** [indrimuska/angular-moment-picker#98]
Hi,

first of all, thanks for creating moment-picker. I was quite frustrated with uib-datepicker and this works great.

I have a small issue with using one of the official examples:
When I use this snippet
```html
<div class="input-group"
     moment-picker="ctrl.datepicker"
     format="YYYY-MM-DD">
    <span class="input-group-addon">
        <i class="octicon octicon-calendar"></i>
    </span>
    <input class="form-control"
           placeholder="Select a date"
           ng-model="ctrl.datepicker"
           ng-model-options="{ updateOn: 'blur' }">
</div>
```
I cannot use my pointer (i.e. trackpad, mouse) to set the cursor in the <input> field. Clicking on it will open the picker and always place the cursor at the end of the string, regardless of where the click occured. If `keyboard="false"` is set, I can navigate using the left and right keys on the keyboard, but my users expect it to behave just as any other input field would. 
Is there any way to prevent this from happening?

**Comment by indrimuska** [comment]
Hi @d53dave, thanks for reporting this.
That's not an expected behavior, I'm going to check if it is bug.

**Comment by indrimuska** [comment]
This should be fixed in [v0.8.2](https://github.com/indrimuska/angular-moment-picker/releases/tag/0.8.2). Feel free to reopen this issue for any other problem :)
